eclass/cmake.eclass

```diff
@@ -768,25 +587,33 @@ _cmake_emake_src_make() {
 
 }
 
-# @FUNCTION: cmake-utils_src_make
+# @FUNCTION: cmake_build
 # @DESCRIPTION:
 # Function for building the package. Automatically detects the build type.
 # All arguments are passed to emake.
-cmake-utils_src_make() {
+cmake_build() {
 	debug-print-function ${FUNCNAME} "$@"
 
 	_cmake_check_build_dir
 	pushd "${BUILD_DIR}" > /dev/null || die
 
-	_cmake_${CMAKE_MAKEFILE_GENERATOR}_src_make "$@"
+	_cmake_${CMAKE_MAKEFILE_GENERATOR}_build "$@"
```

Comment:
Could we use `cmake --build .` here?

Reply:
What's the benefit on giving up our established wrappers?